extensions/ql-vscode/src/config.ts

```diff
@@ -608,3 +608,19 @@ export const CODESPACES_TEMPLATE = new Setting(
 export function isCodespacesTemplate() {
   return !!CODESPACES_TEMPLATE.getValue<boolean>();
 }
+
+/**
+ * The name of the folder where we want to create skeleton wizard QL packs.
+ **/
+const SKELETON_WIZARD_FOLDER = new Setting(
+  "folder",
+  new Setting("skeletonWizard", ROOT_SETTING),
```

Comment:
Since this is a semi-public config option (as in, users will be able to see/edit `codeQL.skeletonWizard.folder` in their JSON settings), can we change the name?

From the users' point of view, this is where "CodeQL: Create Query" drops their query, so perhaps something like `createQuery` (instead of `skeletonWizard`) would be more intuitive? 🤔 ☠️

Reply:
Have renamed. Thanks!